llvm/lib/CodeGen/AsmPrinter/DwarfUnit.h

```diff
@@ -34,6 +34,12 @@ class MCSymbol;
 /// source file.
 class DwarfUnit : public DIEUnit {
 protected:
+  // Label to denote the end of Abbreviation contribution of this compile unit.
```

Comment:
///

Reply:
Yes you are right, I got confused, it is the end of the previous abbreviation contribution, but the start of the current one